hive-metastore/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -169,6 +171,15 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
         tbl.getParameters().put("EXTERNAL", "TRUE"); // using the external table type also requires this
       }
 
+      // If  needed set the required properties on the table to be able to query from Hive
+      if (hiveEngineEnabled(metadata, conf)) {
```

Comment:
I think this should be in the `storageDescriptor` method instead of here in the commit.

Reply:
Done